app/src/main/java/com/github/pedrovgs/effectiveandroidui/TvShowsApplication.java

```diff
@@ -49,6 +49,7 @@ public void onCreate() {
    * color changes to show how it works.
    *
    */
+  @SuppressWarnings("PMD.UnusedPrivateMethod")
```

Comment:
Usually disabling a check rule is a bad practice. In this case, as this private method is designed to be used if other programmers want to activate it, it's ok to disable this rule.

Our common scenario is to avoid having unused private methods, as they are probably dead code.